param/morse-prototype.yaml

```diff
@@ -0,0 +1,38 @@
+sim:
+    ground_truth_orientation: false
+    ground_truth_localization: false
+    ground_truth_altitude: false
+    ground_truth_camera_localization: false
+    ground_truth_roombas: true
+    ground_truth_obstacles: false
+
+    # Because MORSE spits out messages all at the same time, if you want these
+    # to come out at MORSE's update frequency (60Hz by default), you have to
+    # make this a little larger than MORSE's frequency
+    max_roomba_output_freq: 61
+    max_obstacle_output_freq: 61
+
```

Comment:
I believe this is missing some params related to noisy roombas. Not sure how them missing from here will impact the simulator when the senior design drone sim is launched.

Reply:
fixed in 509263d